docs/user-guides/configurations.md

```diff
@@ -134,3 +134,15 @@ If using Iceberg Format，please refer to [Iceberg configurations](https://icebe
 | base.hive.auto-sync-schema-change | true             | Whether synchronize schema changes of Hive Table from HMS                                              |
 | base.hive.auto-sync-data-write    | false            | Whether synchronize data changes of Hive Table from HMS, this should be true when writing to Hive      |
 | base.hive.consistent-write.enabled | true            | To avoid writing dirty data, the files written to the Hive directory will be hidden files and renamed to visible files upon commit. |
+
+### Terminal configurations
+
+Terminal support local and kyuubi, the default is local. If the user uses local to run in the spark local context, you can set the **spark.*** configuration, and if you use kyuubi, you can set the **kyuubi.*** configuration
+
+| Key                      | Default | Description                                             |
+|--------------------------|---------|---------------------------------------------------------|
+| terminal.backend         | local   | terminal backend implement. local, kyuubi are supported |
+| terminal.factory         | -       | session factory implement of terminal                   |
```

Comment:
I don't get this
Could you explain more?

Reply:
**ArcticManagementConf.TERMINAL_SESSION_FACTORY** custom treminal implement